lib/net/ssh/known_hosts/cached.rb

```diff
@@ -73,14 +80,17 @@ def parse_known_hosts(source)
           end
         end
 
-        def add(host, key)
+        def add(host, key, _options = nil)
```

Comment:
I needed to keep support for a third positional parameter for `Hostskeys#add_host_key` https://github.com/net-ssh/net-ssh/blob/54476fa1e88a046014fa986792dbf2c65c5f17db/lib/net/ssh/known_hosts.rb#L24